.github/workflows/docs.yml

```diff
@@ -63,4 +63,5 @@ jobs:
 
 
       - name: Deploy docs
-        run: mike deploy --push $DOCS_VERSION
+        run: mike deploy --push --update-aliases $DOCS_VERSION latest
```

Comment:
Hmm, I'm not so sure this is correct. The value of `$DOCS_VERSION` is `latest` (when running on that branch). Wouldn't that just alias it to itself?